stockflux-securities-master/src/components/Alert.css

```diff
@@ -0,0 +1,11 @@
+.securities-message {
+  font-size: var(--font-xxxlarge);
+}
+
+.error {
```

Comment:
As it stands, any element containing the class `.error` or `.success`, regardless of where in your application will receive this styling. You should make it so that this style only applies based on the existence of the `.alert`/`.securities-alert` class as well:

    .alert.error { ... }

Reply:
Yep good point, now changed